src/test/java/com/exasol/dbbuilder/dialects/exasol/ExasolDatabaseObjectCreationAndDeletionIT.java

```diff
@@ -30,13 +30,15 @@
 
 import com.exasol.containers.ExasolContainer;
 import com.exasol.dbbuilder.dialects.*;
+import com.exasol.dbbuilder.dialects.exasol.udf.UdfScript;
 
 @Tag("integration")
 @Testcontainers
 // [itest->dsn~exasol-object-factory~1]
 class ExasolDatabaseObjectCreationAndDeletionIT extends AbstractDatabaseObjectCreationAndDeletionIT {
     @Container
-    private static final ExasolContainer<? extends ExasolContainer<?>> container = new ExasolContainer<>();
+    private static final ExasolContainer<? extends ExasolContainer<?>> container = new ExasolContainer<>()
+            .withReuse(true);
```

Comment:
why is this needed now?

Reply:
it makes allows you to make tests faster. I just added it because it is possible and a new feature